File: Java/1.java
// 1. Two Sum

class Solution {
    public static int[] twoSum(int[] nums, int target) {
        int[] ret = new int[2];
        for(int j = 0 ; j < nums.length ; j ++){
            ret[0] = j;
            int n = target - nums[j];
            //System.out.println(n);
                for(int i = j + 1 ; i < nums.length ; i++){
                    if(nums[i] == n){
                        ret[1] = i;
                        return ret;
                    }
                }
        }
        return null;
    }
}

File: python/1.py
# 1. Two Sum

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n = len(nums)
        numMap = dict()
        for i in range(n):
            if numMap.__contains__(target - nums[i]):
                return [i, numMap.get(target - nums[i])]
            else:
                numMap[nums[i]] = i

